index.html

```diff
@@ -51,6 +51,171 @@ <h1 class="headline__title"><span class="headline__title--black">Hey there. <br>
+                    </div>
+                </article>
+            </div>
+        </section>
+        <!--About me section-->
+        <section class="about">
+            <div class="about__headline">
+                <h2 class="about__headline--title">About me</h2>
+                <p class="about__headline--text">
+                    I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+                <button class="action-btn">Get my resume</button>
+            </div>
+            <div class="about__card-container">
+                <!--Languages Card-->
+                <article class="about__card">
+                    <div class="about__card--icon">
+                        <img src="./assets/languages_icon.png" alt="">
+                    </div>
+                    <div class="about__card--title">
+                        <h3>Languages</h3>
+                    </div>
+                    <div class="about__card--tags">
+                        <ul>
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </div>
+                </article>
+                <!--Frameworks Card-->
+                <article class="about__card">
+                    <div class="about__card--icon">
+                        <img src="./assets/icon-frameworks.png" alt="">
+                    </div>
+                    <div class="about__card--title">
+                        <h3>Frameworks</h3>
+                    </div>
+                    <div class="about__card--tags">
+                        <ul>
+                            <li>React.js</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSpec</li>
+                            <li>Capybara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+                </article>
+                <!--Skills Card-->
+                <article class="about__card">
+                    <div class="about__card--icon">
+                        <img src="./assets/icon-skills.png" alt="">
+                    </div>
+                    <div class="about__card--title">
+                        <h3>Skills</h3>
+                    </div>
+                    <div class="about__card--tags">
+                        <ul>
+                            <li>Database Management</li>
+                            <li>Version Control</li>
+                            <li>CLI</li>
+                            <li>Web Development</li>
+                            <li>API Design</li>
+                        </ul>
+                    </div>
+                </article>
+            </div>
```

Comment:
- [ ] Well done! It is evident that you have a very high interest in semantics and clean code 👍. However, I believe that these three articles are each a technology category or I should say these articles are a ```list``` of technology categories 😉. In that light, I suggest that you put these articles in a list ie. ul tags make it more semantically sound 👍. Your code might end up looking something like this: 

```sh
<ul class="about__card-container">
                <!--Languages Card-->
                <li>
                <article class="about__card">
                    <div class="about__card--icon">
                        <img src="./assets/languages_icon.png" alt="">
                    </div>
                    <div class="about__card--title">
                        <h3>Languages</h3>
                    </div>
                    <div class="about__card--tags">
                        <ul>
                            <li>JavaScript</li>
                            <li>Ruby</li>
                            <li>HTML</li>
                            <li>CSS</li>
                        </ul>
                    </div>
                </article>
                </li>
                <!--Frameworks Card-->
                <li>
                <article class="about__card">
                    <div class="about__card--icon">
                        <img src="./assets/icon-frameworks.png" alt="">
                    </div>
                    <div class="about__card--title">
                        <h3>Frameworks</h3>
                    </div>
                    <div class="about__card--tags">
                        <ul>
                            <li>React.js</li>
                            <li>Ruby on Rails</li>
                            <li>RSpec</li>
                            <li>Capybara</li>
                            <li>Selenium</li>
                        </ul>
                    </div>
                </article>
                </li>
                <!--Skills Card-->
                <li>
                <article class="about__card">
                    <div class="about__card--icon">
                        <img src="./assets/icon-skills.png" alt="">
                    </div>
                    <div class="about__card--title">
                        <h3>Skills</h3>
                    </div>
                    <div class="about__card--tags">
                        <ul>
                            <li>Database Management</li>
                            <li>Version Control</li>
                            <li>CLI</li>
                            <li>Web Development</li>
                            <li>API Design</li>
                        </ul>
                    </div>
                </article>
                </li>
</ul>
```